tests/test_pex.py

```diff
@@ -253,6 +253,23 @@ def test_pex_run():
       assert fake_stdout.read() == b'hellohello'
 
 
+def test_pex_run_extra_sys_path():
+  with named_temporary_file() as fake_stdout:
+    with temporary_dir() as temp_dir:
+      pex = write_simple_pex(
+        temp_dir,
+        'import sys; sys.stdout.write(":".join(sys.path)); sys.exit(0)'
+      )
+      rc = PEX(pex.path()).run(stdin=None, stdout=fake_stdout, stderr=None, env={
+        'PEX_EXTRA_SYS_PATH': 'extra/syspath/entry1:extra/syspath/entry2'})
+      assert rc == 0
+
+      fake_stdout.seek(0)
+      syspath = fake_stdout.read().split(b':')
+      assert b'extra/syspath/entry1' in syspath
+      assert b'extra/syspath/entry2' in syspath
```

Comment:
Looks good! Could you please add a base case where you do not set the env var to be extra defensive? They can “assert foo not in” for example

Reply:
That seems excessive. What would I test for? That a random string isn't in the sys.path? How would I pick that string?

Reply:
Eh, fair. Looks good